### ISSUE TITLE
Car Object Detection

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#31]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Car Object Detection</br>
:red_circle: **Aim** : Create a DL model which will detect the objects(cars) from the given dataset.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/dataclusterlabs/small-vehicle-images-far-vehicle-detection</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by tejas910** [comment]
hey, am i assigned to this issue?

**Comment by Rushijaviya** [comment]
No, not yet.

**Comment by tejas910** [comment]
Name: Tejas Patil
Github link:- https://github.com/tejas910
Email Id: tppatil910@gmail.com
Approach: To separates car from all other vehicle
role: contributor

**Comment by Rushijaviya** [comment]
Assigned to @tejas910. Happy Contibution!

**Comment by tejas910** [comment]
Thanks

**Comment by abhisheks008** [comment]
@tejas910 updates please.

**Comment by tejas910** [comment]
hi @abhisheks008 work is going on to make the model more accurate is quite a challenging task but I am trying my best sorry for the inconvenience

**Comment by abhisheks008** [comment]
It's okay, thanks for the update. Keep going @tejas910

**Comment by abhisheks008** [comment]
Definitely, it's a deep learning project, use whatever you feel better!

@tejas910

**Comment by abhisheks008** [comment]
Updates please @tejas910
